services/etl/dags/etl_waqi_readings.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+
+import pendulum
+from airflow.decorators import dag, task
+from api_interface import get_waqi_data as gwd
+from db.db_engine import get_db
+from db.models import Readings_WAQI_Temp
+from util.read_sql import read_sql
+
+
+@dag(
+    schedule=timedelta(minutes=1),
+    start_date=pendulum.datetime(2022, 1, 1, tz="UTC"),
+    catchup=False,
+    tags=["readings"],
+)
+def etl_readings_waqi():
+    """
+    This dag retrieves air quality readings data from World Air Quality Index project: https://aqicn.org/api/
+    """
+        
+    @task()
+    def create_temp_waqi():
```

Comment:
so you're using the `get_db()` for everything? does that mean you don't have to set up a connection on the airflow web ui? if so that's dope